packages/container/lib/container.js

```diff
@@ -182,6 +182,10 @@ Container.prototype = {
   }
 })();
 
+function isSingleton(container, fullName) {
+  return container._registry.getOption(fullName, 'singleton') !== false;
```

Comment:
The `singleton` option by default is `true`, so neither alternative above would work for all cases (`true`, `false`, `undefined`). One alternative that would work is `return !!container._registry.getOption(fullName, 'singleton');` - I'm guessing that's what you meant with your first suggestion. Anyway, I'd be glad to change to this if it's more a more common convention throughout ember. There are other instances of checking `!== false` throughout the Container which I could also change to `!!` if there's consensus on this.